build_test/lib/src/package_reader.dart

```diff
@@ -77,7 +77,7 @@ class PackageAssetReader extends AssetReader
   File _resolve(AssetId id) {
     final uri = id.uri;
     if (uri.isScheme('package')) {
-      return new File(_packageResolver.resolveUri(id.uri).path);
+      return new File.fromUri(_packageResolver.resolveUri(id.uri));
```

Comment:
I think you can also do `.filePath`, which is how I fixed this before? Either way probably works though?

Reply:
yeah I don't know if there is much reason to prefer one over the other.

I have a slight preference for `File.fromUri` if only because the type errors are nicer than "you gave `path` but should have given `toFilePath()` and they're both strings"